app/gui/src/config.rs

```diff
@@ -67,7 +72,16 @@ impl BackendService {
                         .namespace
                         .clone()
                         .unwrap_or_else(|| constants::DEFAULT_PROJECT_NAMESPACE.to_owned());
-                    Ok(Self::LanguageServer { json_endpoint, binary_endpoint, namespace })
+                    let project_name = args
+                        .project
+                        .clone()
+                        .ok_or_else(|| MissingOption(args.names().project()))?;
```

Comment:
I have comment about that in other PRs. Please check such places in the code. We prefer multiple variables over multi-line expressions. I've been writing longer comments about that in other places. See the style guide for more info.

Reply:
Apparently it's too easy to skip such places when briefly looking through the code, I fixed it.